CMakeLists.txt

```diff
@@ -36,7 +34,7 @@ set ( Dripline_PYTHON_THROW_REPLY_KEYWORD "ThrowReply" CACHE STRING "Keyword use
 set( Dripline_MAX_PAYLOAD_SIZE "10000" CACHE STRING "Maximum payload size (bytes)" )
 
 # always use C++11
-set_option( USE_CPP11 TRUE )
+set( CMAKE_CXX_STANDARD 11 )
```

Comment:
I have no reason to doubt that this is right, but hadn't there been discussion of moving to a newer standard? maybe 13?

Reply:
There's nothing wrong with using C++14, 17 or 20 for anything that uses dl-cpp.  But there's also no need for it for this particular package.  This is more of a minimum version.  There's not really a way of enforcing the minimum version aspect of it unless I were to put in much more complicated logic, which doesn't seem worth it.  Instead the build would just fail.